### PR TITLE
Set up testing in master/3.2

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

**Comment by VSEphpbb** [comment]
Depends on #16

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - SNIFF="1"        # Should we run code sniffer on your code?
+    - IMAGE_ICC="1"    # Should we run icc profile sniffer on your images?
+    - EPV="1"          # Should we run EPV (Extension Pre Validator) on your code?
+    - PHPBB_BRANCH="master"
+
+branches:
+  only:
+    - master
+    - develop
+    - /^develop-.*$/
```
Comment (michaelcullum) [inline .travis.yml]: erm

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - EXTNAME="phpbb/ideas" # CHANGE name of the extension HERE
+    - SNIFF="1"        # Should we run code sniffer on your code?
+    - IMAGE_ICC="1"    # Should we run icc profile sniffer on your images?
+    - EPV="1"          # Should we run EPV (Extension Pre Validator) on your code?
+    - PHPBB_BRANCH="master"
+
+branches:
+  only:
+    - master
+    - develop
```
Comment (michaelcullum) [inline .travis.yml]: also erm

File: .travis.yml (hunk 1 above)
Comment (VSEphpbb) [inline .travis.yml]: what?

Comment (michaelcullum) [inline .travis.yml]: we don't use that branch naming scheme

**Comment by rxu** [comment]
Looks good. Although I consider ```branches: only:``` statement unhandy as it prevents custom branches from running tests before making a PR.

**Comment by VSEphpbb** [comment]
The branches only mitigates unwanted testing. Without it, if you create a patches fix, that branch will run tests. Then your PR will run another round of the tests again. Only the PR should really run tests, outside of the normal project branches.

**Comment by VSEphpbb** [comment]
@michaelcullum :smile:

**Pull request merged**